script/Ciel/Day 5/05_01_CIEL05_1E.txt

```diff
@@ -243,17 +245,18 @@ I realize the unsightliness of running away from my guilt, of the selfishness of
 [sha:19bfc35924a0f58e5372ceb5a9f8099e0d392711]{
 -- Page 216, Offset 11443.
 -- 自分がろくでなしである事を、俺はもう否定できない。
-I don't deny anymore that I'm a good-for-nothing.
+I can no longer deny that I'm a good-for-nothing.
 }
 [sha:fbf27f1bcdc9f7868b11c0cd7ba6f252f6ae85f1]{
 -- Page 217, Offset 11444.
 -- でも。その上で、俺は俺にできる事をしたいんだ。
-But. I also want to do what I can.
+But, I also want to do whatever I can.
 }
 [sha:75768039b970fb92daf44ac9719cac4a6628b763]{
 -- Page 217, Offset 11445.
 -- ここで逃げ出したら本当に、俺は<無|た><能|だ>の人殺しだ。そんなヤツに、罪の償いができる筈がない。
-If I run away here, then I'll really be <an incompetent|j u s t  a  > murderer. There's no way I'll be able to atone for my sins then.
+// not too sure, can bring back ruby if this one doesnt read well
```

Comment:
I'm kinda indifferent. I don't really think the ruby adds anything and it kinda makes it confusing to read